### DISCUSSION TITLE
mysql json field update

### DISCUSSION THREAD

**Discussion by w-Bro** [sqlalchemy/sqlalchemy#7379] [Usage Questions] [answered]
- i have a db model like this
```
class TaskLog(Base):

    sys_username = Column(VARCHAR(255), nullable=False)
    sys_password = Column(VARCHAR(255), nullable=False")
    params = Column(JSON, nullable=True)
    status = Column(VARCHAR(1), nullable=False)
    task_result = Column(JSON, nullable=True)

```
- i would like to update a value in params with a certain key, just like this  raw sql 
```update openapi_task_log set params=json_set(params, "$.orderNo", "123456")```
- how could i implement it with `Query` object？
```
db_session.query(model).filter(xxx).update(???)
```

**Comment by CaselIT** [comment, marked as answer]
Hi,

does `func` not work?
So something like `update(params=sa.func.json_set(model.params, "$.orderNo", "123456"))`

**Reply by w-Bro** [reply]
thanks, it work.
